Ports/.port_include.sh

```diff
@@ -223,20 +223,28 @@ install_icon() {
 
     command -v magick >/dev/null || true
     local magick_exists=$?
+    command -v convert >/dev/null || true
+    local convert_exists=$?
     command -v identify >/dev/null || true
     local identify_exists=$?
 
-    if [ "${magick_exists}" != "0" ] || [ "${identify_exists}" != 0 ]; then
-        echo 'Unable to install icon: missing magick or identify, did you install ImageMagick?'
+    if [ "${magick_exists}" != "0" ] && [ "${convert_exists}" != 0 ] || [ "${identify_exists}" != 0 ]; then
+        echo 'Unable to install icon: missing magick/convert or identify, did you install ImageMagick?'
         return
     fi
 
+    if [ "${magick_exists}" != 0 ]; then
+        magick_convert=magick
+    else
+        magick_convert=convert
+    fi
```

Comment:
Can we do

```
if [ "${magick_exists}" != 0 ]; then
        magick_convert=magick
elif [ "${convert_exists}" != 0 ]; then
        magick_convert=convert
else
echo 'Unable to install icon: missing magick/convert or identify, did you install ImageMagick?'
        return
    fi
```

in a single conditional, instead of the two you currently have?

Reply:
I still need to check if `identify` exists, but I can get it a bit cleaner.